conf/routes

```diff
@@ -58,8 +58,8 @@ GET    /api/status/:c/topicIdentities                       controllers.api.Kafk
 GET    /api/status/clusters                                 controllers.api.KafkaStateCheck.clusters
 GET    /api/status/:c/:t/underReplicatedPartitions          controllers.api.KafkaStateCheck.underReplicatedPartitions(c:String,t:String)
 GET    /api/status/:c/:t/unavailablePartitions              controllers.api.KafkaStateCheck.unavailablePartitions(c:String,t:String)
-GET    /api/status/:cluster/:consumer/:topic/topicSummary   controllers.api.KafkaStateCheck.topicSummaryAction(cluster:String, consumer:String, topic:String, consumerType:String)
-GET    /api/status/:cluster/:consumer/groupSummary          controllers.api.KafkaStateCheck.groupSummaryAction(cluster:String, consumer:String, consumerType:String)
+GET    /api/status/:cluster/:consumer/:topic/:consumerType/topicSummary   controllers.api.KafkaStateCheck.topicSummaryAction(cluster:String, consumer:String, topic:String, consumerType:String)
```

Comment:
Not sure if we want to change this.  It is backwards incompatible change.  You should just be able to pass in a query parameter for consumerType (e.g. ?consumerType=KF)